### PR TITLE
New Minecraft Chat Cheat Sheet

### PR DESCRIPTION
**Pull request by OutdatedVersion**
Adds a quick lil' cheat sheet

IA Page: https://duck.co/ia/view/minecraft_chat_formatting_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@OutdatedVersion Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/minecraft-chat-formatting.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+    "id": "minecraft_chat_formatting_cheat_sheet",
+    "name": "Minecraft Chat Formatting",
+    "description": "Find those pesky color codes you seem to always forget",
+
+    "metadata": {
+        "sourceName": "Minecraft Wiki",
+        "sourceUrl" : "http://minecraft.gamepedia.com/Formatting_codes"
```
Comment (122131) [inline share/goodie/cheat_sheets/json/minecraft-chat-formatting.json]: Can you please use HTTPS here?

Comment (OutdatedVersion) [inline share/goodie/cheat_sheets/json/minecraft-chat-formatting.json]: I would have if I could. They redirect all HTTPS requests to plain HTTP.

Comment (122131) [inline share/goodie/cheat_sheets/json/minecraft-chat-formatting.json]: They use HTTP only if the user hasn't a Gamepedia PRO account, so I think it's always better to use HTTPS even if it is redirected :+1:

Comment (OutdatedVersion) [inline share/goodie/cheat_sheets/json/minecraft-chat-formatting.json]: There ya go

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=minecraft chat cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/minecraft_chat_formatting_cheat_sheet_mobile.png?cu8sINsQ3M)

**Comment by OutdatedVersion** [comment]
I'm thinking this should be altered to actually be an interactive 'builder' for chat messages. So you can view your messages as they would in-game here instead. Just the cheat sheet is pretty boring.

**Comment by GuiltyDolphin** [comment]
@OutdatedVersion That sounds like a good idea - it'd require a bit of JS; would you like to hold off on this and build a Goodie to support the interactive coloring?

**Comment by OutdatedVersion** [comment]
ya @GuiltyDolphin. Probably end up fairly similar to [this](http://www.thomasvdbulk.nl/minecraft/), but of course just built into the engine.

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**